src/processTargets/targets/ScopeTypeTarget.ts

```diff
@@ -152,13 +152,25 @@ export default class ScopeTypeTarget extends BaseTarget {
 
 function getDelimiter(scopeType: SimpleScopeTypeType): string {
   switch (scopeType) {
-    case "anonymousFunction":
-    case "statement":
-    case "ifStatement":
-      return "\n";
     case "class":
     case "namedFunction":
+    case "ifStatement":
```

Comment:
I'm not sure if I want if statements to have this insertion delimiter, but we can give it a try.

Reply:
I'd be tempted to try it out and if it's annoying we can revisit

Reply:
We should probably make it configurable, as it is kind of a stylistic decision